.github/workflows/integration_test.yml

```diff
@@ -66,21 +66,22 @@ jobs:
           npm install && npm run build && docker build -t sphinxlightning/sphinx-relay .
       - name: Checkout stack
         run: |
-          git clone https://github.com/stakwork/sphinx-stack.git stack
+          git clone -b feat/cln-support https://github.com/stakwork/sphinx-stack.git stack
       - name: give permissions
         working-directory: ./stack
         run: |
           chmod 777 ./bitcoind;    
           chmod 777 -R ./relay;
           chmod 777 -R ./lnd;
           chmod 777 -R ./proxy;
+          chmod 777 -R ./cln;
       - name: Check for NODES.json
         uses: nick-fields/retry@v2
         with:
           timeout_minutes: 10
           max_attempts: 3
           command: |
-            GITACTION_ENV=gitactionenv docker-compose -f ./stack/alts/proxy.yml --project-dir ./stack up -d;
+            GITACTION_ENV=gitactionenv docker-compose -f ./stack/alts/cln-proxy.yml --project-dir ./stack up -d;
```

Comment:
just a question

So is the current test environment 3 LND nodes and 1 proxy node connected to CLN?

Reply:
yes. but for automated tests we should have both LND+Proxy and CLN+Proxy setups (there are two different compose files)